doc/ref_model/chapters/chapter04.md

```diff
@@ -21,25 +24,73 @@ The idea of the infrastructure profiles catalogue is to have a predefined set of
+
+<p align="center"><b>Table 4-1:</b> Flavour Geometry Specification.</p>
 
 <a name="4.1.1"></a>
-### 4.1.1 Storage extensions
+###  4.1.1	Predefined Compute flavours
+The intent of the following flavours list is to be comprehensive and yet effective to cover both IT and NFV workloads. The compute flavours are specified relative to the standardised “large” flavour. The standard “large” flavour configuration consists of 4 vCPUs, 8 GB of RAM and 80 GB of local disk, and the resulting instance will have a management interface of 1 Gbps. The “medium” flavour is half the size of a large and small is half the size of medium. The tiny flavour is a special sized flavour.
+
+>_*Note:*_ Customised (Parameterized) flavours can be used in concession by operators and , if needed, are  created using TOSCA, HEAT templates and/or VIM APIs.
+
+.conf |vCPU ("c") |RAM ("r") |Local Disk ("d") | Managmenet Interface
+-----|------------|----------|-----|-----
+.tiny	|1	|512 MB	|1 GB	|1 Gbps
+.small	|1	|2 GB	|20 GB 	|1 Gbps
+.medium	|2	|4 GB	|40 GB	|1 Gbps
+.large	|4	|8 GB	|80 GB	|1 Gbps
+.2xlarge*	|8	|16 GB	|160 GB	|1 Gbps
+.4xlarge*	|16	|32 GB	|320 GB	|1 Gbps
+
+<p align="center"><b>Table 4-2:</b> Predefined Compute flavours.</p>
+
+> _*These compute flavours are intended to be used for transitional purposes and VNF vendors are expected to consume smaller flavours and adopt micro server’s designs for their VNFs_
+
+<a name="4.2"></a>
+## 4.2 Network Interface Specifications
+
+The network interface specifications extend the flavour customization to specify the network interface “n” followed by the interface bandwidth (in Gbps) and an alphabetic character defining the number of interfaces with that bandwidth; multiple network interface bandwidths, where network interfaces of different bandwidths exist, can be specified by repeating the “n” option.
+```
+<network interface bandwidth option> :: <”n”><number (bandwidth in Gbps)>< # of interfaces of that bandwidth>
+<number of interfaces> :: <”” | “D” | “T” | “Q” | “P” | “H”> 
+where “” represents 1x, “D” 2x, “T” 3x, “Q” 4x, “p” 5x and “H” 6x interfaces of the given bandwidth.
+```
+
+Virtual network interface option	|Description (Bandwidth in Gbps)
+---|---
+n10, n10D, n10T, n10Q, n10P, n10H	|1x 10, 2x 10, 3x 10, 4x 10, 5x 10, 6x 10 Gbps
+n25, n25D, n25T, n25Q, n25P, n25H	|1x 25, 2x 25, 3x 25, 4x 25, 5x 25, 6x 25 Gbps
+n50, n50D, n50T, n50Q, n50P, n50H	|1x 50, 2x 50, 36x 50, 4x 50, 5x 50, 6x 50 Gbps
+
+<p align="center"><b>Table 4-3:</b> Virtual Network Interface Specification Examples.</p>
+
+<a name="4.3"></a>
+##  4.3 Storage Extensions
+Multiplee non-ephemeral storage volumes can be attached to virtual computes (*replace with decided term*) for persistent data storage. Each of those volumes can be configured with the required performance catagories (*storage performance prfiles*).
+***Add comment about CEPH distributed storage. (Potentially create new profile for it).***
```

Comment:
Why is CEPH distribute storage different? It can be the block storage backend behind e.g. Cinder in OpenStack. 

I assume that Object Storage (with Swift or S3 API) is/will be described somewhere else in the document, as it would be an infrastructure service not bound to a specific compute instance.

Reply:
@hdamker, I recommend not to mention any specific backend storage ( CEPH), Just block, file, object storage ( as the type of storage that required from multiple VNF vendor ) and most seen storage is block

Reply:
Yes, I agree - that's how my comment can be resolved: Delete "***Add comment about CEPH distributed storage. (Potentially create new profile for it).***" in 4.3.

Reply:
Great , Thanks

Reply:
Agree Herbert - I took file/object out of that section in a much earlier version for that reason 👍